gulpfile.js

```diff
@@ -85,7 +85,7 @@ gulp.task('vendor', function vendorTask(done) {
 
 // based on https://github.com/gulpjs/gulp/blob/master/docs/recipes/browserify-with-globs.md,
 // except we use the Promise returned by globby, instead of passing it a callback
-gulp.task('scripts', ['lint'], function scriptsTask() {
+gulp.task('scripts', function scriptsTask() {
```

Comment:
This takes the lint step out of file watching builds on my dev machine. Gets run on a commit hook, but I like to see the linting problems come up while I'm working rather than at the end when I commit